### DISCUSSION TITLE
In the same code context, Vim  and Jetbrains plugin's suggestions are not the same.

### DISCUSSION THREAD

**Discussion by qianwch** [community/community#51436] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I don't know whether it is a bug, but at lease it is unexpected. Here is the code context:
```java
public class BTree {
  // Implement a balanced binary tree

}
```
The suggestions are quite different between the plugins.

**Comment by rilysh** [comment]
Those who implemented the plugin for Vim might have a different "approach" than the Jet brain's one. I assume there isn't much difference between them, a slight difference but pointing to the same context is normal.

**Reply by qianwch** [reply]
> Those who implemented the plugin for Vim might have a different "approach" than the Jet brain's one. I assume there isn't much difference between them, a slight difference but pointing to the same context is normal.

It is quite different. In vim the suggestion is much better than in Jetbrains.
